### ISSUE TITLE
Widget doesn't show up in Jupyter Notebook

### ISSUE THREAD

**Issue by Nithanaroy** [tensorflow/fairness-indicators#143]
**System information**
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04): RHEL 7
- Fairness Indicators version: 
```
fairness-indicators                    0.24.0
tensorboard-plugin-fairness-indicators 0.24.0
```
- Python version: 3.7
- Pip version: 20.2
- NPM version: npm@6.14.4
- Jupyter Notebook version: 
```
jupyter                                1.0.0
jupyter-client                         5.2.3
jupyter-console                        6.1.0
jupyter-contrib-core                   0.3.3
jupyter-contrib-nbextensions           0.5.1
jupyter-core                           4.4.0
jupyter-highlight-selected-word        0.2.0
jupyter-latex-envs                     1.4.6
jupyter-nbextensions-configurator      0.4.1
jupyterlab                             2.2.4
jupyterlab-launcher                    0.11.2
jupyterlab-pygments                    0.1.1
jupyterlab-server                      1.2.0
```

**Describe the problem**
```python
tfma.addons.fairness.view.widget_view.render_fairness_indicator(eval_result)
```
does not display any widget in Jupyter Notebook. It shows these errors in the browser console,
```
manager-base.js:273 Could not instantiate widget
        (anonymous) @ manager-base.js:273
        (anonymous) @ manager-base.js:44
        (anonymous) @ manager-base.js:25
        a @ manager-base.js:17
        Promise.then (async)
        u @ manager-base.js:18
        (anonymous) @ manager-base.js:19
        A @ manager-base.js:15
        t._make_model @ manager-base.js:257
        (anonymous) @ manager-base.js:246
        (anonymous) @ manager-base.js:44
        (anonymous) @ manager-base.js:25
        (anonymous) @ manager-base.js:19
        A @ manager-base.js:15
        t.new_model @ manager-base.js:232
        t.handle_comm_open @ manager-base.js:144
        L @ underscore.js:762
        (anonymous) @ underscore.js:775
        (anonymous) @ underscore.js:122
        (anonymous) @ comm.js:89
        Promise.then (async)
        CommManager.comm_open @ comm.js:85
        i @ jquery.min.js:2
        Kernel._handle_iopub_message @ kernel.js:1223
        Kernel._finish_ws_message @ kernel.js:1015
        (anonymous) @ kernel.js:1006
        Promise.then (async)
        Kernel._handle_ws_message @ kernel.js:1006
        i @ jquery.min.js:2

utils.js:119 Error: Could not create a model.
        at utils.js:119
        (anonymous) @ utils.js:119
        Promise.catch (async)
        t.handle_comm_open @ manager-base.js:149
        L @ underscore.js:762
        (anonymous) @ underscore.js:775
        (anonymous) @ underscore.js:122
        (anonymous) @ comm.js:89
        Promise.then (async)
        CommManager.comm_open @ comm.js:85
        i @ jquery.min.js:2
        Kernel._handle_iopub_message @ kernel.js:1223
        Kernel._finish_ws_message @ kernel.js:1015
        (anonymous) @ kernel.js:1006
        Promise.then (async)
        Kernel._handle_ws_message @ kernel.js:1006
        i @ jquery.min.js:2

2kernel.js:1007 Couldn't process kernel message TypeError: Cannot read property 'FairnessIndicatorModel' of undefined
        at manager.js:153
        (anonymous) @ kernel.js:1007
        Promise.catch (async)
        Kernel._handle_ws_message @ kernel.js:1007
        i @ jquery.min.js:2

manager.js:153 Uncaught (in promise) TypeError: Cannot read property 'FairnessIndicatorModel' of undefined
    at manager.js:153
    (anonymous)	@	manager.js:153
    Promise.then (async)		
    t.register_model	@	manager-base.js:208
    (anonymous)	@	manager-base.js:248
    (anonymous)	@	manager-base.js:44
    (anonymous)	@	manager-base.js:25
    (anonymous)	@	manager-base.js:19
    A	@	manager-base.js:15
    t.new_model	@	manager-base.js:232
    t.handle_comm_open	@	manager-base.js:144
    L	@	underscore.js:762
    (anonymous)	@	underscore.js:775
    (anonymous)	@	underscore.js:122
    (anonymous)	@	comm.js:89
    Promise.then (async)		
    CommManager.comm_open	@	comm.js:85
    i	@	jquery.min.js:2
    Kernel._handle_iopub_message	@	kernel.js:1223
    Kernel._finish_ws_message	@	kernel.js:1015
    (anonymous)	@	kernel.js:1006
    Promise.then (async)		
    Kernel._handle_ws_message	@	kernel.js:1006
    i	@	jquery.min.js:2
```

**Provide the exact sequence of commands / steps that you executed before running into the problem**
I was just following an example from TF docs which uses bar_pass_prediction.csv dataset. Below code shows how the config is created from sample data.

```python
# Specify Fairness Indicators in eval_config.
eval_config = text_format.Parse("""
  model_specs {
    prediction_key: 'dnn_bar_pass_prediction',
    label_key: 'pass_bar'
  }
  metrics_specs {
    metrics {class_name: "AUC"}
    metrics {
      class_name: "FairnessIndicators"
      config: '{"thresholds": [0.50, 0.90]}'
    }
  }
  slicing_specs {
    feature_keys: 'race1'
  }
  slicing_specs {}
  """, tfma.EvalConfig())

# Run TensorFlow Model Analysis.
eval_result = tfma.analyze_raw_data(
  data=_LSAT_DF,
  eval_config=eval_config,
  output_path=_DATA_ROOT)
```

**Any other info / logs**
Include any logs or source code that would be helpful to diagnose the problem. If including tracebacks, please include the full traceback. Large logs and files should be attached.
None

**Comment by shuklak13** [comment]
Thanks for creating the issue.

The Fairness Indicators widget not rendering in Jupyter is an issue we're currently working on tracking to fix.

**Comment by fhuanming** [comment]
This should be fixed with FI v0.28.0.
